universal-login-sdk/lib/SdkSigner.ts

```diff
@@ -43,6 +44,7 @@ export class SdkSigner extends ethers.Signer {
       message.value = await transaction.value;
     }
     const {transactionHash} = await this.sdk.execute(message, this.wallet.privateKey);
-    return this.provider.getTransaction(transactionHash);
+    ensure(!!transactionHash, Error, 'Transaction hash is not found in Message Status');
```

Comment:
`ensureNotFalsy(transactionHash, Error, 'Transaction hash is not found in Message Status');`

Reply:
✔️